spec/lib/twingly/url/hasher_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "spec_helper"
+SimpleCov.command_name(File.basename(__FILE__, ".rb"))
```

Comment:
I don't really understand what this is/why this is needed – do you have more info?